build/preprocessing.py

```diff
@@ -0,0 +1,72 @@
+#! python
+
+import yaml
+import sys
+from os import listdir
+from os.path import isfile, join
```

Comment:
injecting content from modules in the global namespace is bad practice in general.
In this case the script is short enough to be somehow OK.